packages/orbit-components/src/InputSelect/index.tsx

```diff
@@ -391,16 +393,19 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
         emptyState
       );
 
+    const isNotReadOnly = isLargeMobile || !readOnly;
```

Comment:
I added this constant to prevent `isLargeMobile || !readOnly` duplication within the whole file.